src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -47,26 +53,15 @@ class AvramRoutes extends BaseEndpoint {
     handleAuthenticatedRequest(request) { userInfo => PongService.pong(userInfo) }
   }
 
-  // TODO: remove this endpoint when we have more meaningful ways to test database queries
-  @ApiMethod(name = "now", httpMethod = "get", path = "now")
-  def now: Now = {
-    // Explicitly use a Future to make sure the implicit ExecutionContext is being used
-    Now(Await.result(Future(fetchTimestampFromDBWithSlick()), Duration.Inf))
+  @ApiMethod(name = "status", httpMethod = "get", path = "status")
+  def status: Status = {
+    //TODO: make this use HealthMoniter
+    inTransaction(_.sqlDBStatus()).map(_ => Status("ok")).head
   }
```

Comment:
Here's a sort-of status endpoint. We want this to use HealthMoniter eventually (or write something similar) but I'm not doing that this PR.

Reply:
HealthMonitor uses background threads (and akka actors) so I'm not sure it'd work as is in GAE standard. We should think about whether we want asynchronous status checks in Avram, and how to implement them ([task queues](https://cloud.google.com/appengine/docs/standard/java/building-app/taskqueues) might be an option?). This should probably be a backlog task -- this current solution is totally fine for now.

Reply:
Yes, I'd like to stay away from Akka.

By the way, threads that run in the background are perfectly fine in GAE. When they say things like "background threads are not allowed", they're talking about the deprecated GAE Background Threads API.

In an email from Ben Menasha (from Google), he said,
> The term "Background Threads" refers to app engine background threads which are currently deprecated : https://cloud.google.com/appengine/docs/standard/java/runtime#background_threads_1  and not to the creation of a new root threadgroup that outlives the request thread.

This info came out of my prodding about DB connection pools.

Reply:
Ah ok - so maybe a threaded (non-Akka) HealthMonitor implementation would work.

Reply:
wow! that really clears up a misinterpretation. i'll post around so others thinking about GAE standard know this is not a limitation.